pkg/apis/metal3/v1alpha1/baremetalhost_types.go

```diff
@@ -118,6 +142,9 @@ type BareMetalHostSpec struct {
 	// How do we connect to the BMC?
 	BMC BMCDetails `json:"bmc,omitempty"`
 
+	// BIOS settings for bare metal server
+	BIOS *BIOSConfig `json:"bios,omitempty"`
```

Comment:
I would like to update BIOS property from `BIOSConfig` to `*BIOSConfig` in order to make it be `nil` in the case of `bios` is not set in BaremetalHost YAML file

Reply:
I think this makes sense

Reply:
Thank you :) 👍